packages/terra-search-field/src/terra-dev-site/test/search-field/DefaultSearchField.test.jsx

```diff
@@ -1,10 +1,9 @@
 import React from 'react';
-import Base from 'terra-base';
 import SearchField from '../../../SearchField';
 
-const locale = document.getElementsByTagName('html')[0].getAttribute('lang');
-
-export default () =>
-  <Base locale={locale}>
+export default () => (
+  <div>
+    <h4>Extraneous Clickable Text</h4>
```

Comment:
why are we adding `<h4>Extraneous Clickable Text</h4>` to the test examples?

Reply:
Clicking on it removes the blinking mouse cursor from the `<input>` element which can tend to ruin tests.

Reply:
Lets add something like [this](https://github.com/cerner/terra-core/blob/master/packages/terra-form-select/tests/wdio/select-spec.js#L175) to set the cursor to be transparent. We will want to capture the focus states of the elements during screenshots and adding this extra text is not desirable.
```
        browser.execute(() => {
          // Removes the blinking cursor to prevent screenshot mismatches.
          document.querySelector('input').style.caretColor = 'transparent';
        });
```